workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -114,6 +142,8 @@ describe('DataSetService', () => {
             storageName: mockDataSetStorageName,
             externalEndpoints: [mockExistingEndpointName]
           };
+        } else if (id === mockInvalidId) {
```

Comment:
is it possible to move this logic to specific `describe` sections for each test context so its clear which mock results belong to a specific test?

Reply:
fun fact, I had things all separated out long long ago and was asked to push it all up front. I don't think I have time to unwind that today.